src/widget/wtrackmenu.cpp

```diff
@@ -835,13 +843,8 @@ TrackPointer WTrackMenu::getFirstTrackPointer() const {
             // Skip unavailable tracks
         }
         return TrackPointer();
```

Comment:
Replace with `DEBUG_ASSERT(!m_pTrack);` and continue?

Reply:
I don't think so, it's ok for this function to return an empty trackpointer if there is no "first" trackpointer.  In the case that the trackmodel is set, we shouldn't "fall back" to m_ptrack if there is nothing selected in the model, we should return nothing.